rotkehlchen/chain/ethereum/names.py

```diff
@@ -46,7 +45,8 @@ def find_ens_mappings(
                 if cur_time - last_update > ENS_UPDATE_INTERVAL:
                     addresses_to_query.append(address)
                 elif has_name:
-                    ens_mappings[cached_value.address] = cached_value.name  # type: ignore  # mypy doesn't see `isinstance` check  # noqa: E501
+                    ens_mappings[
+                        cached_value.address] = cached_value.name  # type: ignore  # mypy doesn't see `isinstance` check  # noqa: E501
```

Comment:
we never break lines like that. And since you already got the noqa: E501 this is doubly not needed

```suggestion
                    ens_mappings[cached_value.address] = cached_value.name  # type: ignore  # mypy doesn't see `isinstance` check  # noqa: E501
```